app/actions/device/index.js

```diff
@@ -52,7 +47,6 @@ export function setDeviceAsTablet() {
 }
 
 export default {
-    connection,
```

Comment:
`connection` gets used in few key controlling components like `screens/channel/index.js` and `network_indicator/index.js` and passed down to their children. Not needed there any more?